CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Get setup for GitHub
+
+1. If you are new to GitHub, don't start here. Instead, work through a GitHub tutorial such as one of these:
+    * https://guides.github.com/activities/hello-world/
+    * http://r-pkgs.had.co.nz/git.html.
+2. Follow [the instructions](https://github.com/all-of-us/workbench#git-secrets) to install [git-secrets](https://github.com/awslabs/git-secrets). Its a git commit hook that *"Prevents you from committing passwords and other sensitive information to a git repository."*.
+
+# How to contribute a snippet
+
+If you want to add/modify a snippet that uses a dataframe from Dataset Builder as its input, then see [dataset-snippets/README](./dataset-snippets/README.md).
+
+Otherwise, see [snippets/README](./snippets/README.md).
+
+# Auto-generation of Jupyter 'Snippets Menu' configuration.
```

Comment:
Optional: it may make sense to make a note about how this makes it into the Workbench environment.

Namely that we need to trigger a manual process to pull in snippets updates for deployment. Once this happens, new snippets contents go live in users' Jupyter servers on a rolling basis over a period of a few weeks.